docs/migration_cleanup.md

```diff
@@ -105,7 +105,7 @@ drush --uri=$DRUSH_OPTIONS_URI search-api:index default_solr_index
 
 4. Rebuild the OAI-PMH cache
 ```bash
-drush --uri=$DRUSH_OPTIONS_URI idr:roai
+drush --user=1 --uri=$DRUSH_OPTIONS_URI idr:roai
```

Comment:
```suggestion
drush --uri=$DRUSH_OPTIONS_URI idr:roai
```

This command doesn't support `--user`. Reasoning here is that OAI entries should only be generated with the anonymous user context given you would not want potentially locked down entries to show as available for harvest.